dist/20.10.7.sh

```diff
@@ -226,6 +226,9 @@ check_forked() {
 				fi
 				dist_version="$(sed 's/\/.*//' /etc/debian_version | sed 's/\..*//')"
 				case "$dist_version" in
+					11)
+						dist_version="bullseye"
+					;;
 					10)
```

Comment:
Hello @Spiegie thanks for the PR.
The scrips under `dist/` is auto generated. You have to do the updates under `pkg/` and then do `make generate` to generate the script under `dist/`. Please take a look at https://github.com/rancher/install-docker#previous-manual-instructions-to-add-a-new-version

Reply:
Thank you very much. Sorry that I neglected my research. Thank you for pointing me in the right direction.